material-ui/lab strict dependenecy

npm v8 has very strict peer dependency requirements. What is in your `package.json` must match what's in the peer requirements. Currently the peer requirements for @material/lab is `^4.0.0` which doesn't exist (and may never since they're moving to `mui`) The peer requirement needs to be specific for the alpha version like so `"@material-ui/lab" : "^4.0.0-alpha.60"` so that it can resolve a version. If I put `"@material-ui/lab": "^4.0.0"` I get an error about it not existing, and if I try the alpha version I get this error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: marketplace_next_playground@0.1.0
npm ERR! Found: @material-ui/lab@4.0.0-alpha.60
npm ERR! node_modules/@material-ui/lab
npm ERR!   @material-ui/lab@"^4.0.0-alpha.60" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @material-ui/lab@"^4.0.0" from @prenda/spark@0.14.0
npm ERR! node_modules/@prenda/spark
npm ERR!   @prenda/spark@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See <path>.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     <path>/2021-10-20T22_39_56_130Z-debug.log
```

I'll create a PR shortly to solve this.

--- a/libs/spark/package.json
+++ b/libs/spark/package.json
@@ -14,6 +14,6 @@
   "peerDependencies": {
     "react": "^16.8.0 || ^17.0.0",
     "@material-ui/core": "^4.0.0",
-    "@material-ui/lab": "^4.0.0"
+    "@material-ui/lab": "^4.0.0-alpha.60"
   }
 }

